XR_APILAYER_MBUCCHIA_toolkit/layer.cpp

```diff
@@ -275,7 +275,9 @@ namespace {
             // not being advertised. However for certain apps using double-wide rendering, we can still enable the
             // feature, based on the list of apps below.
             m_overrideFoveatedRenderingCapability =
-                m_isOpenComposite && m_applicationName == "OpenComposite_AC2-Win64-Shipping";
```

Comment:
checking `m_isOpenComposite` is redundant and removed,  because  `m_isOpenComposite` is just a check if the `m_applicationName` starts with OpenComposite_

Reply:
that's fine yes